app/src/main/java/com/wesaphzt/privatelocation/service/LocationService.java

```diff
@@ -182,7 +182,7 @@ private void pushLocation(Intent intent) {
             if (intent.hasExtra("lat") && intent.hasExtra("lng") ) {
                 double lat = intent.getDoubleExtra("lat", 45);
                 double lng = intent.getDoubleExtra("lng", 45);
-
+                
```

Comment:
I guess this can reverted.

Reply:
I don't see the point. Why should there be a commented out section be left behind?

Reply:
Ah, I thought your comment refers to the deleted lines, but I if you just refer to the empty spaces in the empty line I will remove them

Reply:
Yep.. 👍🏿